src/Compilers/Core/Portable/Compilation/CompilationContext.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+
+namespace Microsoft.CodeAnalysis.Collections
```

Comment:
Should CompilationContext be in the Collection namespace? Its name doesn't quite sound as a collection.